packages/node_modules/@webex/plugin-meetings/src/meeting/util.js

```diff
@@ -18,10 +22,39 @@ MeetingUtil.parseLocus = (response) => {
   return parsed;
 };
 
+MeetingUtil.remoteUpdateAudioVideo = (audio, video, meeting) => {
+  if (meeting) {
+    const localMedias = Media.generateLocalMedias(meeting.mediaId, audio, video);
+    if (isEmpty(localMedias)) {
+      return Promise.reject(new Error('You need a media id on the meeting to change remote audio.'));
+    }
+    return meeting.meetingRequest.remoteAudioVideoToggle({
+      locusUrl: meeting.locusUrl,
+      selfId: meeting.selfId,
+      localMedias,
+      deviceUrl: meeting.deviceUrl,
+      correlationId: meeting.correlationId
+    });
+  }
+  return Promise.reject(new Error('You need a meeting object.'));
+};
+
+MeetingUtil.leaveMeeting = (meeting) => {
+  if (meeting) {
+    return meeting.meetingRequest.leaveMeeting({
+      locusUrl: meeting.locusUrl,
+      selfId: meeting.selfId,
+      correlationId: meeting.correlationId,
+      resourceId: null,
+      deviceUrl: meeting.deviceUrl
+    });
+  }
+  return Promise.reject(new Error('You need a meeting object.'));
```

Comment:
I'd really like to "return/fail early" in these scenarios. I think it's more than personal preference in this case but we can check with the sdk team if you disagree. 

```
if (!meeting) {
  return Promise.reject()
}

return meeting.meetingRequest...

Reply:
they are equivalent, but we can do that.